Dockerfile

```diff
@@ -15,7 +15,7 @@ LABEL "maintainer"="Matt Brown <github@muglug.com>"
 
 RUN apk add --no-cache tini git
 
-COPY --from=composer:1.9 /usr/bin/composer /usr/bin/composer
+COPY --from=composer:$INPUT_COMPOSER_VERSION /usr/bin/composer /usr/bin/composer
```

Comment:
not sure if this is how we can access the input from github action ?

Reply:
Nevermind, as the image will be built once I think and not on the fly according to the args of the action.